en/peripherals/dshot.md

```diff
@@ -0,0 +1,79 @@
+# DShot ESCs
+
+DShot is an alternative ESC protocol that has several advantages over PWM or OneShot:
+- Reduced latency
+- Checksum (increased robustness)
+- Digital encoding (no need for ESC calibrations)
+- Possibility to get feedback (telemetry)
+- Sending commands (such as reversing the motor spin direction)
+
+DShot ESCs are connected and wired the same way as [PWM ESCs](pwm_escs_and_servo.md), and you can switch between these protocols just by changing software parameters (ESCs automatically detect the selected protocol on startup).
+
+## Configuration
+
+> **Warning** Remove propellers before changing ESC configuration parameters!
+
+Enable DShot with the [DSHOT_CONFIG](../advanced_config/parameter_reference.md#DSHOT_CONFIG) parameter (if the parameter does not exist, the board does not support DShot).
+
+DShot comes with different speed options: *DShot150*, *DShot300*, *DShot600* and *DShot1200*, where the number indicates the speed in bits per second.
+You should set the parameter to the highest speed supported by your ESC (according to its datasheet) and then reboot the vehicle.
+
+Then connect the battery and arm the vehicle.
+The ESCs should initialize and the motors turn.
+If the motors do not spin at lowest throttle, increase [DSHOT_MIN](../advanced_config/parameter_reference.md#DSHOT_MIN) until they spin.
+
+> **Tip** On boards with an IO, DShot is only supported on the AUX pins.
+> It is best to disable the IO via [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and then connect the motors to the AUX pins instead of MAIN.
```

Comment:
On boards with IO Dshot is only supported on the AUX ports (FMU). This tips suggests that we disable the IO board and connect to MAIN (Same physical ports on FMU), but it isn't obvious why.

What I think (?) you are saying is that DSHOT only works on FMU pins, so if you want to map outputs to motors as per the airframe reference you're going to have to disable the IO so that those pins are MAIN. Correct?

Further implication being that anything mapped to AUX ports will have to be unavailable if you use DSHOT ESC.

Reply:
> What I think (?) you are saying is that DSHOT only works on FMU pins, so if you want to map outputs to motors as per the airframe reference you're going to have to disable the IO so that those pins are MAIN. Correct?

Correct.

> Further implication being that anything mapped to AUX ports will have to be unavailable if you use DSHOT ESC.

Right.

Reply:
OK, so to address this I've put wiring into its own section and mentioned the issue there, and more clearly stated that you have to make the change in the configuration section. Please check.